gdmix-workflow/test/resources/detext-movieLens.yaml

```diff
@@ -11,30 +11,33 @@
       "num_filters": 50,
       "num_train_steps": 1000,
       "num_units": 64,
-      "optimizer": "bert_adam",
+      "optimizer": "adamw",
```

Comment:
why changed the optimizer?

Reply:
DeText updated the optimizer options: https://github.com/linkedin/detext/blob/master/src/detext/args.py#L330